src/components/common/TmBalance.vue

```diff
@@ -346,14 +345,28 @@ export default {
       )
     },
     totalRewardsPerDenom() {
-      if (this.overview.rewards && this.overview.rewards.length > 0) {
-        return this.overview.rewards.reduce((all, reward) => {
+      if (this.rewards && this.rewards.length > 0) {
+        return this.rewards.reduce((all, reward) => {
           return {
             ...all,
             [reward.denom]: parseFloat(reward.amount) + (all[reward.denom] || 0)
           }
         }, {})
       } else return {}
+    },
+    totalRewards() {
+      if (this.rewards && this.rewards.length > 0) {
+        const totalRewards = this.rewards
+          .filter(({ denom }) => denom === this.stakingDenom)
+          .reduce((sum, { amount }) => parseFloat(amount) + sum, 0)
+          .toFixed(6)
+        if (!this.rewardsSentToGA) {
+          this.sendRewards(totalRewards)
+        }
+
+        return totalRewards
+      }
+      return 0
```

Comment:
```suggestion
      return this.totalRewardsPerDenom[this.stakingDenom] || 0
```

Reply:
you can move
```
if (!this.rewardsSentToGA) {
          this.sendRewards(totalRewards)
        }
```
to a watcher or inside `totalRewardsPerDenom`

Reply:
Done